Test/Machine/layer_input_tests.hpp

```diff
@@ -40,6 +40,18 @@ std::vector<netket::json> GetLayerInputs() {
           {"Hamiltonian", {{"Name", "Heisenberg"}}}};
   input_tests.push_back(pars);
 
+  // Recurrent layer
+  pars = {{"Graph",
```

Comment:
@kchoo1118 where is the sigma of the random parameters set in the testing? You mentioned it is sigma=1 by default?

Reply:
line 109 in unit-machine.cc